cfct/template/vpc_flowlog_logging_bucket.template

```diff
@@ -0,0 +1,116 @@
+# This is a sample, non-production-ready template
+# © 2021 Amazon Web Services, Inc. or its affiliates. All Rights Reserved.  
+# This AWS Content is provided subject to the terms of the AWS Customer Agreement available at  
+# http://aws.amazon.com/agreement or other written agreement between Customer and either
+# Amazon Web Services, Inc. or Amazon Web Services EMEA SARL or both.
+
+AWSTemplateFormatVersion: 2010-09-09
+Description: Create a S3 storage bucket in the audit account.
+
+Parameters:
+  OrganizationId:
+    Type: 'String'
+    Description: ID of the AWS Organization of the Landing Zone
+  SSEAlgorithm:
+    Type: 'String'
+    Default: 'AES256'
+    Description: S3 bucket SSE Algorithm.
+    AllowedValues:
+    - 'AES256'
+    - 'aws:kms'
+  KMSMasterKeyID:
+    Type: 'String'
+    Description: 'KMS key ID required if SSE algorithm is aws:kms.'
+
+Conditions:
+  UseKMS: !Equals
+    - !Ref SSEAlgorithm
+    - 'aws:kms'
+  UseAES256: !Equals
+    - !Ref SSEAlgorithm
+    - 'AES256'
+
+Resources:
+
+  S3BucketFlowLoggingSSE:
+    Type: AWS::S3::Bucket
+    Condition: UseAES256
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:

```

Comment:
bucket needs to have versioning, ref [cb9424b](https://github.com/aws-samples/aws-ct-vpc-flowlogs/pull/7/commits/cb9424b557ddad6d9a33a284f737868b5c88ebdf)

Reply:
done